From 6e8614607db757ae25858430063d29e6b33f0d05 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 4 Sep 2022 22:34:56 +0200
Subject: [PATCH 03/24] arm64: dts: rk3399: Disable debug nodes

They hang boot with older levinboot, due to levinboot not enabling their PD.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-base.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-base.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-base.dtsi
index 4dcceb9136b7..867473ff05ee 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-base.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-base.dtsi
@@ -479,6 +479,7 @@ usb_host1_ohci: usb@fe3e0000 {
 		status = "disabled";
 	};
 
+	/*
 	debug@fe430000 {
 		compatible = "arm,coresight-cpu-debug", "arm,primecell";
 		reg = <0 0xfe430000 0 0x1000>;
@@ -526,6 +527,7 @@ debug@fe710000 {
 		clock-names = "apb_pclk";
 		cpu = <&cpu_b1>;
 	};
+        */
 
 	usbdrd3_0: usb@fe800000 {
 		compatible = "rockchip,rk3399-dwc3";
-- 
2.43.0

